truckersmp_cli/utils.py

```diff
@@ -353,6 +353,42 @@ def get_short_size(size_bytes):
     return "{:.1f}M".format(size_bytes / 1048576)
 
 
+def get_steam_library_dirs(steamdir):
+    """
+    Get Steam library directories.
+
+    steamdir: Steam installation path
+    """
+    # the first Steam library directory is the Steam installation
+    steam_libraries = [steamdir, ]
+    # additional directories are stored in libraryfolders.vdf
+    #
+    # example:
+    #     "LibraryFolders"
+    #     {
+    #         "TimeNextStatsReport"  "xxxx"
+    #         "ContentStatsID"       "xxxx"
+    #         "1"                    "/path/to/steam/library1"
+    #         "2"                    "/path/to/steam/library2"
+    #     }  -> [steamdir, "/path/to/steam/library1", "/path/to/steam/library2"]
+    try:
+        with open(os.path.join(steamdir, File.steamlibvdf_inner)) as f_vdf:
```

Comment:
Any particular reason you aren't using python-vdf here?

Reply:
Yes.

* We don't need to *parse VDF*
    * It's enough to read each line and check the 2 quoted string
    * `get_steam_library_dirs()` is already simple
* I would not like to add a "required" dependency
    * `get_steam_library_dirs()` would be a part of "basic" feature